widgetastic_manageiq/__init__.py

```diff
@@ -4547,6 +4547,32 @@ def data_for_timestamp(self, timestamp):
         return tooltip_data
 
 
+class ReactTextInput(TextInput):
+
+    # The clear method from the WebElement class in the Selenium package was not clearing the react
+    # text field. arguments[0] is the widget name
+    clear_text_field = """\
+        (function(elem){
+            var elem_name = document.getElementsByName(elem);
+            var elem_id = elem_name[0].id;
+            document.getElementById(elem_id).value = "";
+        }(arguments[0]));
+    """
+
+    def clear(self):
```

Comment:
So I think there's a good enhancement here for `Widgetastic.widget.input.TextInput`.

Defining `TextInput.clear`, and modifying its fill implementation to use `self.clear` instead of `self.browser.clear` directly would allow for subclassing widgets to define their own clear mechanisms, without having to re-define `TextInput.fill` locally.

This would need to be done in wt.core, and this PR isn't blocked by this - but it would allow for a slimmer implementation here.